### DISCUSSION TITLE
edit css in exported html file.

### DISCUSSION THREAD

**Discussion by jlop77** [Lumieducation/Lumi#1986] [Q&A]
Hi,

I'm testing. I export h5p file to html and it works fine, but I want to: set a maximum width, change the font size, because it looks too small on the smartphone.

How can I do this? I'm a newbie.

**Comment by sr258** [comment]
Hi,

this is not possible at the moment. We've ideas about adding this feature, but we're not there yet.

Best
Sebastian
